test/runTest.m

```diff
@@ -1,6 +1,29 @@
+%==========================================================================
+%This function runs the testFeatures tests, which test specific features of
+%the viewer, and the testDataSets tests, which test all datasets available in the
+%file.
+%
+%Authors: Thomas Maullin, Camille Maumet.
+%==========================================================================
+
 function runTest()
+    
+    %Setup steps for the test.
```

Comment:
One improvement would be to add the `lib` folder to the matlabpath automatically if missing. This would avoid the following error:
```
>> addpath('PATH/TO/spm12')
>> addpath('PATH/TO/nidmresults-spmhtml/')
>> addpath('PATH/TO/nidmresults-spmhtml/test/')
>> runTest
Undefined function 'searchforType' for input arguments of type
'cell'.

Error in createTest (line 46)
        [designMatrix, dmLocation] =
        searchforType('nidm_DesignMatrix', json.x_graph);

Error in runTest (line 15)
    createTest();
```